Library/Formula/terraform.rb

```diff
@@ -63,6 +63,9 @@ def install
 
     cd terrapath do
       terraform_files = `go list ./...`.lines.map { |f| f.strip unless f.include? "/vendor/" }.compact
+      # v0.6.12 - source contains tests which fail if these environment variables are set locally.
+      ENV["AWS_ACCESS_KEY"] = ""
```

Comment:
Rather than setting blank strings you can use `ENV.delete "AWS_ACCESS_KEY"` and such.